src/module/actor/SR5Actor.ts

```diff
@@ -2294,4 +2296,41 @@ export class SR5Actor<SubType extends Actor.ConfiguredSubType = Actor.Configured
         await StorageFlow.deleteStorageReferences(this);
         return super._preDelete(...args);
     }
+
+    async deleteItem(item: SR5Item | SR5ActiveEffect) {
```

Comment:
why also `SR5ActiveEffect`? I was unable to find a reference for this function. is this a handler?